src/es/ikigaimangas/src/eu/kanade/tachiyomi/extension/es/ikigaimangas/IkigaiMangas.kt

```diff
@@ -173,7 +195,46 @@ class IkigaiMangas : HttpSource(), ConfigurableSource {
         apiUrl.addQueryParameter("column", sortByFilter?.selected ?: "name")
         apiUrl.addQueryParameter("direction", if (sortByFilter?.state?.ascending == true) "asc" else "desc")
 
-        return GET(apiUrl.build(), lazyHeaders)
+        return GET(apiUrl.build(), headers)
+    }
+
+    val scriptUrlRegex = """from"(.*?.js)"""".toRegex()
```

Comment:
`.*?.` is effectively `.+?`, you probably meant:
```suggestion
    val scriptUrlRegex = """from"(.*?\.js)"""".toRegex()
```